lib/linguist/languages.yml

```diff
@@ -5580,6 +5580,15 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+vvvv:
+  type: markup
+  color: "#C0C0C0"
+  extensions:
+  - ".v4p"
+  - ".vl"
+  tm_scope: none
```

Comment:
`source.xml` would be better than nothing here.

Reply:
Ok, I'll change that

Reply:
`source.xml` doesn't seem to exist in `grammars.yml`. What should I use instead? `text.xml`?

Reply:
Yes, my bad.